nats.go

```diff
@@ -2507,8 +2507,10 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
-// Servers returns the list of known servers, including additional
-// servers discovered after a connection has been established.
+// Servers returns the list of known server urls, including additional
+// servers discovered after a connection has been established.  URLs passed
+// into NATS through the API be preserved in their original form, but any
+// credentials in a URL will never be visible outside of the application.
```

Comment:
That being said, it would be very easy to return: "nats://host:port" for all urls in the server pool. The thing I don't know is what is the use case of this new API. If the user is going to use those to issue a connect, then getting the full URL (possibly with credentials) will help. The user can use UserInfo() and/or Token() to help with the URLs that would not have credentials.